pyproject.toml

```diff
@@ -1,7 +1,111 @@
 [build-system]
-requires = ["setuptools>=40.8.0", "setuptools_scm[toml]>=6.2", "Cython", "py-cpuinfo"]
-build-backend = "setuptools.build_meta:__legacy__"
+requires = [
+    "setuptools>=64",
+    "setuptools-scm[toml]>=6.2",
+    "Cython",
+    "py-cpuinfo"
+]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "numcodecs"
+description = """
+A Python package providing buffer compression and transformation codecs for use
+in data storage and communication applications.
+"""
+readme =  "README.rst"
+dependencies = [
+    "entrypoints",
+    "numpy>=1.7",
+    "typing-extensions>=3.7.4"
+]
+requires-python = ">=3.7"
+dynamic = [
+  "version",
+]
+classifiers = [
+    "Development Status :: 4 - Beta",
```

Comment:
Perhaps outwith this PR but is "Beta" still appropriate?

Reply:
Good question. Filed as issue ( https://github.com/zarr-developers/numcodecs/issues/394 )